front-end/src/lighting/main.jsx

```diff
@@ -196,6 +187,21 @@ class main extends React.Component {
     }
   }
 
+  getModeLabel (mode) {
+    let label = ''
```

Comment:
what if you remove the variable initialization and break statement all together, and just return on every case
```js
switch(mode) {
case 'auto':
  return i18n.t(...)
case '':
 return ...
} 
```